src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -197,6 +199,9 @@ export const portfolioApi = createApi({
           const portfolioAccounts = { [pubkey]: await adapter.getAccount(pubkey) }
           const nftCollectionsById = selectNftCollections(state)
 
+          const maybePortalsAccounts = await fetchPortalsAccount(chainId, pubkey)
+          const maybePortalsPlatforms = await fetchPortalsPlatforms()
```

Comment:
we can move this inside the IIFE to only fetch for evm chains, throwing errors as it stands trying to fetch utxo and cosmossdk chains

Reply:
9256068b8b